### ISSUE TITLE
Failed to find Node 20.13.0 (LTS)

### ISSUE THREAD

**Issue by htunnicliff** [asdf-vm/asdf-nodejs#391]
I am unable to install the latest LTS version of Node (v20.13.0) which was released on 2024-05-07.

Here is the output I'm seeing:

```sh
$ asdf install nodejs 20.13.0
Trying to update node-build... ok
node-build: definition not found: 20.13.0
```

I've attempted to update node-build using a variety of commands, none of which result in the new version becoming available.

Any ideas if this is an asdf issue, node-build issue, or something else entirely?

**Comment by chedabob** [comment]
I suspect it's because this PR hasn't been merged https://github.com/nodenv/node-build/pull/866

**Comment by dzienisz** [comment]
I can't understand why I can't just write asdf install nodejs 20 for newest 20.X version.

**Comment by augustobmoura** [comment]
> I am unable to install the latest LTS version of Node (v20.13.0) which was released on 2024-05-07.
> 
> Here is the output I'm seeing:
> 
> ```shell
> $ asdf install nodejs 20.13.0
> Trying to update node-build... ok
> node-build: definition not found: 20.13.0
> ```
> 
> I've attempted to update node-build using a variety of commands, none of which result in the new version becoming available.
> 
> Any ideas if this is an asdf issue, node-build issue, or something else entirely?

> I suspect it's because this PR hasn't been merged [nodenv/node-build#866](https://github.com/nodenv/node-build/pull/866)

Exactly, just give some time between new releases so definitions are merged to node-build

> I can't understand why I can't just write asdf install nodejs 20 for newest 20.X version.

That syntax is not supported by `asdf` core, instead you can write:
```
asdf install nodejs latest:20
```
